jetstream_pt/layers.py

```diff
@@ -156,10 +155,7 @@ def __call__(self, xq, xk, xv, mask, cache):
     with jax.named_scope("attn_mat1"):
       ## Attention start
       # scores = torch.einsum(jnp.einsum, "ijkl,ikml->ikjm", xq, keys) / math.sqrt(self.head_dim)
-      scores = torch_xla2.extra.call_jax(
-          jnp.einsum, "ikjl,ikml->ikjm", xq, keys
-      ) / math.sqrt(head_dim)
-      self.env.apply_sharding(scores, axis=1)
+      scores = torch.einsum("ikjl,ikml->ikjm", xq, keys) / math.sqrt(head_dim)
```

Comment:
Is b/329899712 fixed?

Reply:
yes (hence the updating the version of torch_xla2)